src/wallet/wallet.cpp

```diff
@@ -4658,3 +4712,78 @@ std::shared_ptr<CWalletTransactionBase> CWalletTransactionBase::MakeWalletObject
         return std::shared_ptr<CWalletTransactionBase>( new CWalletTx(pwallet, dynamic_cast<const CTransaction&>(obj)) );
     }
 }
+
+/* Find unspent notes filtered by payment address, min depth and max depth */
+void CWallet::GetUnspentFilteredNotes(
+    std::vector<CUnspentNotePlaintextEntry>& outEntries,
+    std::set<PaymentAddress>& filterAddresses,
+    int minDepth,
+    int maxDepth,
+    bool requireSpendingKey)
+{
```

Comment:
This opening block adds an unneeded indentation level : the lock stays active till end or early exit of function